exporter.go

```diff
@@ -34,22 +34,23 @@ type Exporter struct {
 	scrapeMySQLConnectionPool      bool
 	scrapeMySQLConnectionList      bool
 	scrapeDetailedMySQLProcessList bool
+	scrapeMemoryMetrics            bool
 	scrapesTotal                   prometheus.Counter
 	scrapeErrorsTotal              *prometheus.CounterVec
 	lastScrapeError                prometheus.Gauge
 	lastScrapeDurationSeconds      prometheus.Gauge
 	proxysqlUp                     prometheus.Gauge
 }
 
-// NewExporter returns a new ProxySQL exporter for the provided DSN.
-// It scrapes stats_mysql_global, stats_mysql_connection_pool and stats_mysql_processlist if corresponding parameters are true.
-func NewExporter(dsn string, scrapeMySQLGlobal bool, scrapeMySQLConnectionPool bool, scrapeMySQLConnectionList bool, scrapeDetailedMySQLProcessList bool) *Exporter {
+// NewExporter It scrapes stats_mysql_global and stats_mysql_connection_pool if corresponding parameters are true.
```

Comment:
I think there was a merge conflict on that line. The first line is lost, and the second one is outdated.

Reply:
Ehh you're correct. Nice spot.

Reply:
Fixed.